.vib/gitea/cypress/cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "username": "user@example.com",
```

Comment:
I think it is better to use a non-default value for this param. Sorry to repeat myself 😁